### PR TITLE
ZebraTechnologiesCorporation.ZebraDesigner version 2.5.0.9420

### PR DESCRIPTION
**Pull request by upintheairsheep**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] This PR only modifies one (1) manifest
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/107474)

### CODE REVIEW

**Comment by stephengillie** [comment]
Manual Validation encountered: 
image

File: manifests/z/ZebraTechnologiesCorporation/ZebraDesigner/2.5.0.9420/ZebraTechnologiesCorporation.ZebraDesigner.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: ZebraTechnologiesCorporation.ZebraDesigner
+PackageVersion: 2.5.0.9420
+Installers:
+- InstallerUrl: http://ftp.ics.cz/Zebra/Designer/zebradesigner-v250-9420.exe
+  Architecture: x86
+  InstallerType: exe
+  InstallerSha256: FAE8DD644EB426C0DC394A50FB1BC1E238451180BFB2A4E13B82ED87DDC77F74
```
Comment (stephengillie) [inline manifests/z/ZebraTechnologiesCorporation/ZebraDesigner/2.5.0.9420/ZebraTechnologiesCorporation.ZebraDesigner.installer.yaml:10]:
```suggestion
  InstallerSha256: FAE8DD644EB426C0DC394A50FB1BC1E238451180BFB2A4E13B82ED87DDC77F74
  InstallerSwitches:
    Silent: /s
```

Comment (stephengillie) [reply manifests/z/ZebraTechnologiesCorporation/ZebraDesigner/2.5.0.9420/ZebraTechnologiesCorporation.ZebraDesigner.installer.yaml:10]: This package uses the InstallAware Wizard, which uses the `/s` switch for [silent install](https://www.installaware.com/forums/viewtopic.php?t=4442#:~:text=To%20do%20this%20you%20can%20go%20to%20Start,%3Csetup%3E%20%2Fs%20this%20will%20run%20the%20installer%20silently.). I do not know if there is a silent-with-progress switch.

**Comment by stephengillie** [comment]
Using the silent install switch above, the package installs normally, and asks for a printer on first run: 
image

File: manifests/z/ZebraTechnologiesCorporation/ZebraDesigner/2.5.0.9420/ZebraTechnologiesCorporation.ZebraDesigner.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: ZebraTechnologiesCorporation.ZebraDesigner
+PackageVersion: 2.5.0.9420
+Installers:
+- InstallerUrl: http://ftp.ics.cz/Zebra/Designer/zebradesigner-v250-9420.exe
```
Comment (stephengillie) [inline manifests/z/ZebraTechnologiesCorporation/ZebraDesigner/2.5.0.9420/ZebraTechnologiesCorporation.ZebraDesigner.installer.yaml:7]:
```suggestion
- InstallerUrl: https://www.zebra.com/apps/dlmanager?dlp=7b8a235653193b4c72c440110c25661656f56f5180957c98e7c0bc2144149cd156a1bc6e684725abae8eaa3b64ee10902b6e77e74d8082bcecf0ffbe929cfe5e5810126dd5b9b88a5fcc3c78383a5eaf7ad78c349cbcb259f5555a613f845da395cb92691dcef46172cce19429d4f28cc2105744ba5f4ca7&c=us&l=en
```

Comment (stephengillie) [reply manifests/z/ZebraTechnologiesCorporation/ZebraDesigner/2.5.0.9420/ZebraTechnologiesCorporation.ZebraDesigner.installer.yaml:7]: I'm not sure that this URL will work for other users, as it appears to use a unique key. Also the website is designed to require EULA agreement before processing the download for older versions. For the current version 3.0 there appear to be fewer download protections.

**Comment by upintheairsheep** [comment]
Add a software dependancy tag. Even with the printer connected it will likely error out still.

**Comment by upintheairsheep** [comment]
Fix hash

**Comment by ItzLevvie** [comment]
[Policy] Interactive-Only-Installer

**Pull request closed without merging**